<!-- THE BUG0 ALGORITHM IMPLEMENTED FOR TURTLEBOT3 -->
<launch>
    <arg name="x_goal" default="2.0"/>
    <arg name="y_goal" default="0.0"/>
    <include file="$(find turtlebot3_gazebo)/launch/turtlebot3_stage_4.launch"/>
    <node name="follow_wall_server" pkg="bug0" type="follow_wall.py" output="screen"/>
    <node name="go_to_goal_server" pkg="bug0" type="go_to_goal.py" output="screen"/>
    <node name="bug0" pkg="bug0" type="main.py" output="screen">
        <param name="x_goal" value="$(arg x_goal)" type="string"/>
        <param name="y_goal" value="$(arg y_goal)" type="string"/>
    </node>
</launch>
